### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by leiserfg**
**IMPORTANT: Please do not create a Pull Request adding a new feature without discussing it first.**

The place to discuss new features is the forum: <https://zola.discourse.group/>
If you want to add a new feature, please open a thread there first in the feature requests section.

Sanity check:

* [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?

## Code changes
(Delete or ignore this section for documentation changes)

* [ ] Are you doing the PR on the `next` branch?

If the change is a new feature or adding to/changing an existing one:

* [ ] Have you created/updated the relevant documentation page(s)?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM bitnami/minideb AS builder
+RUN install_packages python-pip curl tar python-setuptools rsync binutils
+RUN pip install dockerize
+RUN mkdir -p /workdir
+WORKDIR /workdir
+ENV DOCKER_TAG v0.7.0
+RUN curl -L https://github.com/getzola/zola/releases/download/$DOCKER_TAG/zola-$DOCKER_TAG-x86_64-unknown-linux-gnu.tar.gz | tar xz
```
Comment (Keats) [inline Dockerfile:7]: how does that work with the dockerhub automatic build?

Comment (leiserfg) [reply Dockerfile:7]: 1. Create a repo on dockerhub and connect your github account.
2.  Go to the build settings (easy way, append /builds/edit to the dockerhub repository url)
3. Use this settings 
cloud docker com_repository_docker_leiserfg_zola_builds_edit

Comment (Keats) [reply Dockerfile:7]: But in that dockerfile the version is hardcoded?

**Comment by leiserfg** [comment]
Is just a default value, it will use the one from the environment.

**Comment by Keats** [comment]
Ok I have setup everything I think in dockerhub, we'll see if it works for the next release. Thanks!

**Pull request merged**